codemp/rd-dedicated/G2_API.cpp

```diff
@@ -649,7 +642,7 @@ int G2API_InitGhoul2Model(CGhoul2Info_v **ghoul2Ptr, const char *fileName, int m
 
 qboolean G2API_SetLodBias(CGhoul2Info *ghlInfo, int lodBias)
 {
-	if (ghlInfo)
+	if (G2_SetupModelPointers(ghlInfo))
```

Comment:
Why are these changed?

Reply:
To be in sync with the original code and rd-vanilla.